.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
+
 php:
-    - 5.5
-    - 5.4
+    - 7.2
+    - 7.3
+    - 7.4
 
-before_script:
-    - composer selfupdate
-    - composer install --prefer-source
+env:
+    matrix:
+        - COMPOSER_FLAGS="--prefer-lowest"
+        - COMPOSER_FLAGS=""
 
-script: phpunit
```

Comment:
This line here, I meant

Reply:
Oh, ok. 
It is strange to me.  Early it was failed on this line. I think, that I fix the path here. But all tests are running, I check Travis build log.
I think, it is some magic and Travis running script section in composer, but I cannot find anything about it now. If you want, I fix this line in travis.yml config.